suite-native/firmware/src/components/FirmwareInstallationScreenContent.tsx

```diff
@@ -267,7 +280,10 @@ export const FirmwareInstallationScreenContent = ({
         >
             <Box flex={1}>
                 <VStack justifyContent="center" alignItems="center" flex={1}>
-                    <UpdateProgressIndicator progress={progress} status={indicatorStatus} />
+                    <UpdateProgressIndicator
+                        progress={indicatorProgress}
+                        status={indicatorStatus}
```

Comment:
Would swap the order as `status` is the primary one.
```suggestion
                        status={indicatorStatus}
                        progress={indicatorProgress}
```